server/src/routes/api/v1/pizzaPlaceReviewRouter.js

```diff
@@ -0,0 +1,24 @@
+import express from "express"
+import { ValidationError } from "objection"
+
+import { Review } from "../../../models/index.js"
+import cleanUserInput from "../../../services/cleanUserInput.js"
+
+const pizzaPlaceReviewRouter = new express.Router({ mergeParams: true })
+
+pizzaPlaceReviewRouter.post("/", async (req, res) => {
+  const body = req.body
+  body.pizzaPlaceId = req.params.id
+  const formInput = cleanUserInput(body)
+  try {
+    const newPizzaReview = await Review.query().insertAndFetch(formInput)
```

Comment:
the back-end here is where we can access the currently signed in user!

https://learn.launchacademy.com/teams/admins/curricula/on-campus-boston-39/lesson_groups/weeks_7_&_8:_group_project/lessons/authentication-with-react-and-passport